### PR TITLE
[DRAFT] Edited using colors in styled components using our Theme (issue#1410)

### PR DESCRIPTION
**Pull request by Dekshut**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing applications:)

**What changes did you make?** (Give an overview)
Edited colors formats in styled components using the Theme, also added new gray colors object to the Colors and some new Theme`s objects.
**Is there anything you'd like reviewers to focus on?**
Edited some tests to include testing the ThemeProvider component

**How Has This Been Tested?** (put an "X" next to an item)

- [ ] No need to
- [X] Manually(please, describe, when necessary)
- [ ] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "X" next to an item, otherwise PR will fail)
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [X] My changes generate no new warnings(e.g. Sonar is happy)
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

### CODE REVIEW

File: kafka-ui-react-app/src/components/Schemas/Details/__test__/LatestVersionItem.spec.tsx [hunk 1]
```diff
@@ -27,7 +37,11 @@ describe('LatestVersionItem', () => {
 
   it('matches snapshot', () => {
     expect(
-      shallow(<LatestVersionItem schema={jsonSchema} />)
+      shallow(
```
Comment (NelyDavtyan) [inline kafka-ui-react-app/src/components/Schemas/Details/__test__/LatestVersionItem.spec.tsx:40]: Oleg told us that we have to replace the enzyme with a react-testing-library.

File: kafka-ui-react-app/src/components/common/ConfirmationModal/ConfirmationModal.styled.tsx [hunk 2]
```diff
@@ -1,58 +1,60 @@
...
+        font-size: 20px;
+        text-align: start;
+      }
 
-    & > section {
-      border-top: 1px solid ${Colors.neutral[5]};
-      border-bottom: 1px solid ${Colors.neutral[5]};
-    }
+      & > section {
+        border-top: 1px solid ${Colors.neutral[5]};
```
Comment (NelyDavtyan) [inline kafka-ui-react-app/src/components/common/ConfirmationModal/ConfirmationModal.styled.tsx:48]: Why didn't you take it from theme?

File: kafka-ui-react-app/src/components/Schemas/Details/__test__/LatestVersionItem.spec.tsx (hunk 1 above)
Comment (workshur) [reply kafka-ui-react-app/src/components/Schemas/Details/__test__/LatestVersionItem.spec.tsx:40]: @NelyDavtyan is right. But @Dekshut never mind for now it is enough

File: kafka-ui-react-app/src/components/common/ConfirmationModal/ConfirmationModal.styled.tsx (hunk 2 above)
Comment (workshur) [reply kafka-ui-react-app/src/components/common/ConfirmationModal/ConfirmationModal.styled.tsx:48]: @Zorii4 will fix it in his own PR

File: kafka-ui-react-app/src/components/common/Icons/CloseIcon.tsx [hunk 3]
```diff
@@ -1,6 +1,8 @@
 import React from 'react';
+import { useTheme } from 'styled-components';
 
-const CloseIcon = () => {
+const CloseIcon: React.FC = () => {
+  const theme = useTheme();
```
Comment (workshur) [inline kafka-ui-react-app/src/components/common/Icons/CloseIcon.tsx:5]: 👍

File: kafka-ui-react-app/src/theme/theme.ts [hunk 4]
```diff
@@ -40,6 +41,12 @@ export const Colors = {
     '30': '#90caf9',
     '40': '#64b5f6',
   },
+  gray: {
+    '10': '#f9fafa',
+    '80': 'rgba(34, 41, 47, 0.5)',
+    '85': 'rgba(10, 10, 10, 0.1)',
+    '90': 'rgba(0, 0, 0, 0.1)',
+  },
```
Comment (workshur) [inline kafka-ui-react-app/src/theme/theme.ts:49]: I think it should be a part of `neutral`

**Review by workshur — COMMENTED** [review]
Looks good so far

File: kafka-ui-react-app/src/components/common/ConfirmationModal/ConfirmationModal.styled.tsx (hunk 2 above)
Comment (Dekshut) [reply kafka-ui-react-app/src/components/common/ConfirmationModal/ConfirmationModal.styled.tsx:48]: Bcs @Zorii4 should be do this fix, it`s not a my bussiness

File: kafka-ui-react-app/src/theme/theme.ts (hunk 4 above)
Comment (Dekshut) [reply kafka-ui-react-app/src/theme/theme.ts:49]: Is`t `neutral` filled enough?

Comment (Dekshut) [reply kafka-ui-react-app/src/theme/theme.ts:49]: Is`t neutral filled enough?

Comment (Hurenka) [reply kafka-ui-react-app/src/theme/theme.ts:49]: agree with workshur

**Review by Hurenka — APPROVED** [review]

**Pull request closed without merging**